src/data/collection.ts

```diff
@@ -17,12 +17,12 @@ interface OptimisticCollection<T> extends Collection<T> {
  */
 
 export const getOptimisticCollection = <StateType>(
-  saveCollection: (conn: Connection, data: StateType) => Promise<unknown>,
+  saveCollection: (conn2: Connection, data: StateType) => Promise<unknown>,
```

Comment:
I don't understand this change, the variable name is not shadowing anything ?

Reply:
True, it is not needed for this one because it is above `conn`, but feels weird not do this one and do the others...

Reply:
I don't _really_ care. `conn2` is just a little weird, maybe `conn_` would be better.